src/pybind.cpp

```diff
@@ -9,6 +9,8 @@
 
 namespace py = pybind11;
 extern int register_server(unsigned long loop_ptr, server_config_t config);
+extern int set_log_level(std::string level);
+extern void printing(std::string level, std::string msg);
```

Comment:
maybe we could have better name?

Reply:
I am not good at naming... could you suggest a name?